src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -413,6 +449,83 @@ void CodeGen::genHWIntrinsic_R_R_RM_I(GenTreeHWIntrinsic* node, instruction ins)
     }
 }
 
+void CodeGen::genJumpTableFallback(
+    GenTreeHWIntrinsic* node, regNumber targetReg, regNumber op1Reg, regNumber op2Reg, regNumber op3Reg)
+{
+    assert(op1Reg != REG_NA);
+    assert(op2Reg != REG_NA);
+    NamedIntrinsic  intrinsicID    = node->gtHWIntrinsicId;
+    var_types       baseType       = node->gtSIMDBaseType;
+    emitAttr        attr           = (emitAttr)node->gtSIMDSize;
+    HWIntrinsicFlag flags          = Compiler::flagsOfHWIntrinsic(intrinsicID);
+    instruction     ins            = Compiler::insOfHWIntrinsic(intrinsicID, baseType);
+    int             numArg         = Compiler::numArgsOfHWIntrinsic(intrinsicID);
+    emitter*        emit           = getEmitter();
+    regNumber       nonConstImmReg = op3Reg == REG_NA ? op2Reg : op3Reg;
+
+    assert((flags & HW_Flag_NoJmpTableIMM) == 0);
+    assert(numArg >= 2);
+
+    unsigned    jmpCount = Compiler::immUpperBoundOfHWIntrinsic(intrinsicID, baseType) + 1;
+    BasicBlock* jmpTable[256];
+
+    unsigned jmpTableBase = emit->emitBBTableDataGenBeg(jmpCount, true);
+    unsigned jmpTableOffs = 0;
+
+    // Emit the jump table
+
+    JITDUMP("\n      J_M%03u_DS%02u LABEL   DWORD\n", Compiler::s_compMethodsCount, jmpTableBase);
```

Comment:
I don't think this is really useful. It looks like it was copied from `genJumpTable` but then it didn't belong there either. This kind of dump messages, if really needed, should be in `emitBBTableDataGenBeg` and `emitDataGenData`.

Or better, the data section should be included in the disassembly produced by the JIT. But that's a different story.